keyboards/suikagiken/suika27melo/keymaps/default/keymap.json

```diff
@@ -0,0 +1,12 @@
+{
+  "author": "@suikagiken",
+  "keyboard": "suikagiken/suika85ergo",
```

Comment:
This value is wrong; the board does not compile properly in the Configurator. Please fix this.